components/normalizer/src/lib.rs

```diff
@@ -67,6 +67,18 @@
 
 extern crate alloc;
 
+// We don't depend on icu_properties to minimize deps, but we want to be able
+// to ensure we're using the right CCC values
+macro_rules! ccc {
+    ($name:ident, $num:expr) => {{
+        #[cfg(all(debug_assertions, feature = "icu_properties"))]
+        if icu_properties::CanonicalCombiningClass::$name.0 != $num {
+            panic!("icu_normalizer has incorrect ccc values")
```

Comment:
Praise: A static assertion?! Since when has that worked?

Reply:
we've been using panicking in consts for years, that's how all the `icu::locale` macros work.